extensions/wikia/Chat2/ChatAjax.class.php

```diff
@@ -262,6 +262,8 @@ static public function giveChatMod() {
 	private static function authenticateServer() {
 		global $wgRequest;
 
+		\Wikia\Security\CSRFDetector::markRequestAsSecure( __METHOD__ );
```

Comment:
this is used in a following way:
`authenticateServer() or authenticateUser()`
which means that if it is a request from user in this specific code we won't actually check if the method was correct because it will be flagged as accepted while doing server authentication check
so I still think we should not mark method as accepted here unconditionally